app/bundles/EmailBundle/Tests/Model/EmailModelTranslationCountFunctionalTest.php

```diff
@@ -0,0 +1,211 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\EmailBundle\Tests\Model;
+
+use Mautic\CampaignBundle\Entity\Campaign;
+use Mautic\CampaignBundle\Entity\Event;
+use Mautic\CampaignBundle\Entity\Lead as CampaignLead;
+use Mautic\CoreBundle\Test\MauticMysqlTestCase;
+use Mautic\EmailBundle\Entity\Email;
+use Mautic\EmailBundle\Model\EmailModel;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\Model\LeadModel;
+
+class EmailModelTranslationCountFunctionalTest extends MauticMysqlTestCase
```

Comment:
All new classes should be final by default
```suggestion
final class EmailModelTranslationCountFunctionalTest extends MauticMysqlTestCase
```

Reply:
I always forgot to add the final keyword 😟